yarn-project/foundation/src/abi/abi.ts

```diff
@@ -267,6 +268,34 @@ export type DebugFileMap = Record<
   }
 >;
 
+/**
+ * Type representing a note in use in the contract.
+ */
+export type ContractNote = {
+  /**
+   * Note identifier
+   */
+  id: Fr;
+  /**
+   * Type of the note
```

Comment:
```suggestion
   * Type of the note (e.g. `ValueNote`)
```
Is my suggestion correct? If yes would be helpful to have it committed